functions-methods.js

```diff
@@ -33,4 +54,21 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(eMailAddress) {
+    const containsAtSign = eMailAddress.includes("@");
+    const containsComma = eMailAddress.includes(",");
+    const finalChacacter = eMailAddress.charAt(eMailAddress.length - 1);
+
+    if (containsAtSign && !containsComma && finalChacacter !== ".") {
+        return true;
+    }
+    else {
+        return false;
+    }
```

Comment:
Dit kun je nog korter schrijven 😉 
```suggestion
    return containsAtSign && !containsComma && finalChacacter !== ".";
```